styles.css

```
@@ -1,5 +1,383 @@
+        4.2 Introduction
+        4.3 Work
+        4.4 Sevices
+        4.5 Content
+        4.6 Footer
+     5. Media Queries
+
+////////////////////////////////////////////////////////////////////////*/
+
+
+/*////////////////////////
+    1. Global Styles
+////////////////////////*/
 * {
     box-sizing: border-box;
     padding: 0;
     margin: 0;
 }
+
+body {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-weight: 300;
+    font-size: 20px;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-grey: #F3F6F9;
+    --grey: #7A7A7A
+}
+
+/*////////////////////////
+    2. Typography
+////////////////////////*/
+h1, h2, h3, h4, h5, h6 {
+    font-family: Merriweather, 'Times New Roman', serif;
+}
+h1, h2 {
+    font-size: 46px;
+    padding: 16px 0;
+}
+
+h3 {
+    font-size: 36px;
+}
+
+h4 {
+    font-size: 32px;
+    padding: 12px 0;
+    margin: 24px 0;
+}
+
+h5 {
+    font-size: 32px;
+    padding: 12px 0;
+}
+
+h6 {
+    font-size: 32px;
+    line-height: 1.2;
+    padding: 12px 0;
+}
+
+p {
+    margin: 12px 0;
+}
+
+/*////////////////////////
+    3. Reusables
+////////////////////////*/
+.outer-content-container {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-content-container {
+    flex-basis: 1400px;
+    flex-shrink: 1;
+    flex-grow: 0;
+    margin: 0 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.text-restrictor {
+    max-width: 850px;
+    text-align: center;
+}
+
+.area-padding {
+    padding: 100px 0;
+}
+
+a,
+a:visited,
+a:link {
+    text-decoration: none;
+    color: #5722CD;
+}
+
+button {
+    font-size: 14px;
+    font-weight: 300;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    color: white;
+    background-color: #ED4385;
+    border: none;
+    border-radius: 50px;
+    box-shadow: 0 5px 8px 4px rgba(0, 0, 0, .2);
+    padding: 25px 40px;
+    margin: 28px 0;
+}
+
+button:hover {
+    cursor: pointer;
+}
+
+
+/*////////////////////////
+    4. Areas
+////////////////////////*/
+/*///////////// 4.1 Header /////////////*/
+
+#header {
+     background: var(--blue);
+     background: linear-gradient(129deg, var(--blue) 0%, var(--blue) 26%, var(--purple)  100%);
+     color: white;
+     padding: 30px 0;
+     position: relative;
+ }
+
+.navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+}
+
+.navigation ul {
+    flex-basis: 400px;
+    flex-shrink: 1;
+    flex-grow: 0;
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.navigation a {
+    color: white;
+}
+
+.work-together {
+    padding: 100px 0;
+    margin-bottom: 88px;
+    font-size: 28px;
+}
+
+.header__squiggle {
+    display: block;
+    width: 78px;
+    height: 100px;
+    margin: 60px 0 0 0;
```

Als je geen margin-right, -bottom en -left hebt om te overschrijven, dan prefereer ik margin-top
```suggestion
    margin-top: 60px;
```